interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
```

Comment:
It is part of the core snap and interacts quite heavily with the kernel, so not sure if we should do that.